controllers/cluster_controller_test_test.go

```diff
@@ -67,7 +65,7 @@ func TestClusterReconcilerEnsureOwnerReferences(t *testing.T) {
 	cb := fake.NewClientBuilder()
 	cl := cb.WithRuntimeObjects(objs...).Build()
 
-	r := controllers.NewClusterReconciler(cl, nullLog(), cl.Scheme(), nil, nil, newDummyProviderReconcilerBuilder())
+	r := controllers.NewNewClusterReconciler(cl, nullLog(), newRegistryForDummyProviderReconciler())
```

Comment:
What is the purpose of the `_test_test` file?

Reply:
The original `_test` file is in the `controllers` package as opposed to `controllers_test`

This file was added to be able to add tests under `controllers_test` until the old tests get moved to the proper test package